apps/web/src/app/api/checker/cron/_cron.ts

```diff
@@ -44,6 +46,7 @@ export const cron = async ({
       url: `${DEFAULT_URL}/api/checker/regions/${region}`,
       body: {
         url: `${DEFAULT_URL}/api/ping`,
+        cronTimestamp: timestamp, // used to group all region requests - can be also cronId
```

Comment:
We will need to pass some sort of identifier to track which calls belong together as the individual `timestamp` will have different values. Can be also done via `cronId`.